packages/solutions/app-tools/src/analyze/nestedRoutes.ts

```diff
@@ -10,11 +10,37 @@ const replaceDynamicPath = (routePath: string) => {
   return routePath.replace(/\[(.*?)\]/g, ':$1');
 };
 
+const getPathWithoutExt = (filename: string) => {
+  const extname = path.extname(filename);
+  return filename.slice(0, -extname.length);
+};
+
+export const getRouteId = (
```

Comment:
why move this package from `@modern-js/utils`

Reply:
only app-tools use this util